synaptic_reconstruction/inference/util.py

```diff
@@ -44,14 +44,14 @@ def get_prediction(
     Returns:
         The predicted volume.
     """
-    # We always use the same default halo.
-    halo = {"x": 64, "y": 64, "z": 16}
+    # make sure either model path or model is passed
+    if model is None and model_path is None:
```

Comment:
I moved this from the vesicle segmentation function here, so that the same patter of passing either `model` or `model_path` could be used in other segmentation functions without duplicating this check.